shared/common-adapters/icon.constants.js

```diff
@@ -0,0 +1,265 @@
+  'icon-hacker-news-logo-24': true,
+  'icon-hacker-news-logo-32': true,
+  'icon-hacker-news-logo-48': true,
+  'icon-hacker-news-logo-64': true,
+  'icon-history-clear-48': true,
+  'icon-history-clear-64': true,
+  'icon-invite-code-48': true,
+  'icon-invite-code-no-more-48': true,
+  'icon-invite-link-24': true,
+  'icon-invite-link-32': true,
+  'icon-invite-link-negative-48': true,
+  'icon-invite-link-negative-64': true,
+  'icon-keybase-logo-128': true,
+  'icon-keybase-logo-16': true,
+  'icon-keybase-logo-160': true,
+  'icon-keybase-logo-24': true,
+  'icon-keybase-logo-256': true,
+  'icon-keybase-logo-32': true,
+  'icon-keybase-logo-48': true,
+  'icon-keybase-logo-64': true,
+  'icon-no-gpg': true,
+  'icon-paper-key-32': true,
+  'icon-paper-key-48': true,
+  'icon-paper-key-64': true,
+  'icon-paper-key-corner': true,
+  'icon-paper-key-dark-blue-32': true,
+  'icon-paper-key-revoke-48': true,
+  'icon-paper-key-revoke-64': true,
+  'icon-paper-key-success-48': true,
+  'icon-people-add-32': true,
+  'icon-pgp-key-16': true,
+  'icon-pgp-key-24': true,
+  'icon-pgp-key-32': true,
+  'icon-pgp-key-48': true,
+  'icon-pgp-key-64': true,
+  'icon-phone-bw-24': true,
+  'icon-phone-bw-32': true,
+  'icon-phone-bw-48': true,
+  'icon-phone-bw-64': true,
+  'icon-phone-bw-revoke-48': true,
+  'icon-phone-bw-revoke-64': true,
+  'icon-phone-bw-success-48': true,
+  'icon-phone-colors-24': true,
+  'icon-phone-colors-32': true,
+  'icon-phone-colors-48': true,
+  'icon-phone-colors-64': true,
+  'icon-phone-dark-blue-32': true,
+  'icon-phone-qr-code-32': true,
+  'icon-phone-qr-code-48': true,
+  'icon-phone-text-code-32': true,
+  'icon-phone-text-code-48': true,
+  'icon-placeholder-avatar-112-x-112': true,
+  'icon-placeholder-avatar-176-x-176': true,
+  'icon-placeholder-avatar-24-x-24': true,
+  'icon-placeholder-avatar-32-x-32': true,
+  'icon-placeholder-avatar-48-x-48': true,
+  'icon-placeholder-avatar-64-x-64': true,
+  'icon-placeholder-avatar-80-x-80': true,
+  'icon-progress-grey-animated': true,
+  'icon-progress-grey-static': true,
+  'icon-progress-white-animated': true,
+  'icon-progress-white-static': true,
+  'icon-reddit-logo-16': true,
+  'icon-reddit-logo-24': true,
+  'icon-reddit-logo-32': true,
+  'icon-reddit-logo-48': true,
+  'icon-reddit-logo-64': true,
+  'icon-subnav-folders-all': true,
+  'icon-subnav-folders-private': true,
+  'icon-subnav-folders-public': true,
+  'icon-terminal-32': true,
+  'icon-terminal-48': true,
+  'icon-twitter-logo-16': true,
+  'icon-twitter-logo-24': true,
+  'icon-twitter-logo-32': true,
+  'icon-twitter-logo-48': true,
+  'icon-twitter-logo-64': true,
+  'icon-update-64': true,
+  'icon-update-pause-64': true,
+  'icon-website-16': true,
+  'icon-website-24': true,
+  'icon-website-32': true,
+  'icon-website-48': true,
+  'icon-website-64': true}
+
+export type IconsType = $Enum<typeof Icons>
+
+export const fontSizes: {[key: string]: number} = {
```

Comment:
flow complains cause we call this with non-FontIconsType (we just pass all the icons to it) and see if the value is undefined